.github/workflows/build.yml

```diff
@@ -101,7 +101,6 @@ jobs:
 
       - uses: ./coordinator/.github/actions/git_tag_release_version
       - uses: ./coordinator/.github/actions/update_next_version
-      - uses: ./coordinator/.github/actions/prepare_release_content
 
```

Comment:
not that I don't 100% trust you but could you add a couple of lines explaining that this has been tested and it is fine? (assuming it has)